lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -735,8 +735,11 @@ def combine_multiple_fields(field_prefix: str) -> Optional[List]:
     record['state'] = combine_multiple_fields('state')
 
     # Age Ceiling
-    record['age'] = age_ceiling(int(record['age']))
-    record['age_months'] = age_ceiling(int(record['age_months']) / 12) * 12
+    try:
+        record['age'] = age_ceiling(int(record['age']))
+        record['age_months'] = age_ceiling(int(record['age_months']) / 12) * 12
+    except ValueError:
+        record['age'] = record['age_months'] = None
```

Comment:
do we not want to fill `age_months` as None, too?

Reply:
@kairstenfay It is assigning both `age` and `age_months` to `None` 😄

Reply:
Oops I read that too quickly. Indeed it is.